api.web.yaml

```diff
@@ -23,166 +24,202 @@ paths:
         default:
           description: Unexpected error
 
-  /profile:
+  # User Management (Admin)
+  /users:
     get:
-      description: Users profile
+      operationId: getAllUsers
```

Comment:
I don't like this overriding. you need to remember both the path and what it's called